sdk/test-utils/perfstress/run.ps1

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
+
+docker run -it --rm --network host system-perfstress-net @args
```

Comment:
I'm not currently using this file, but it will be relevant for: https://github.com/azure/azure-sdk-for-js/issues/8063